common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -78,10 +77,21 @@
 
         Sequence.prototype.updatePageTitle = function() {
             // update the page title to include the current section
-            var positionLink = this.link_for(this.position);
+            var positionLink = this.link_for(this.position),
+                currentSectionTitle;
```

Comment:
@ormsbee nope! The coffeescript file no longer exists, as of this PR: https://github.com/edx/edx-platform/pull/14167